### ISSUE TITLE
Bots names doubles

### ISSUE THREAD

**Issue by gary00k** [yapb/yapb#353]
I noticed, sometimes adding bots to the server causes the new bot enters with the same name one of bots on server already has.
Can you add some function that will check if drawn bot name is already in use on server?

**Comment by gary00k** [comment]
Yup, nothing changed in names.cfg.

**Comment by gary00k** [comment]
Ref?

**Comment by Vladislav4KZ** [comment]
@gary00k are you using the ``yb_save_bots_names 1`` cvar? I noticed that the same bug was reported in the discussions https://github.com/yapb/yapb/discussions/403

There, a person wrote that the bug was reproduced using the cvar ``yb_save_bots_names 1``.

I'll try to figure this out.

**Comment by Vladislav4KZ** [comment]
It seems to be fixed by the commit: https://github.com/yapb/yapb/commit/9d8318e5846fc35b9c99921b8de2005f8d3e717d

I reproduced this bug yesterday on a previous version of the bot, and saw that one or two bots appeared with duplicate nicknames.

To do this, I created a server with 9 bots, then changed the map through the changelevel, waited for these 9 bots to join, then added new bots until I saw duplicate nicknames.

Today, after checking the latest build from GitHub Actions, this bug is no longer reproduced.
